config/src/config/network_config.rs

```diff
@@ -41,7 +41,9 @@ pub const MAX_CONCURRENT_NETWORK_REQS: usize = 100;
 pub const MAX_CONNECTION_DELAY_MS: u64 = 60_000; /* 1 minute */
 pub const MAX_FULLNODE_OUTBOUND_CONNECTIONS: usize = 4;
 pub const MAX_INBOUND_CONNECTIONS: usize = 100;
-pub const MAX_FRAME_SIZE: usize = 16 * 1024 * 1024; /* 16 MiB */
+pub const MAX_FRAME_SIZE: usize = 64 * 1024 * 1024; /* 64 MB: the hard limit for messages on the wire */
```

Comment:
we probably should call it max_message_size

Reply:
Okay, I can clean this up when I approach the mempool changes (as it'll rely on these constants) 😄